File: gem5/src/REU/SimObjects/cache_info.cc
#include "REU/SimObjects/cache_info.hh"
#include "mem/ruby/system/RubySystem.hh"
#include "mem/ruby/structures/CacheMemory.hh"
#include "mem/ruby/network/garnet/Router.hh"


#include "base/trace.hh"
#include "debug/CacheInfoFlag.hh"


namespace gem5
{

CacheInfo::CacheInfo(const Params &params):
    SimObject(params)
{
    DPRINTF(CacheInfoFlag, "%s: Hello World! From a "
                    "SimObject (constructor).\n", __func__);
}

void 
CacheInfo::getNeighbourID() {
    // int id = get_id();

}



} // namespace gem5

File: gem5/src/REU/SimObjects/CacheInfo.py
from m5.params import *
from m5.proxy import *
from m5.SimObject import SimObject

class CacheInfo(SimObject):
    type = "CacheInfo"
    cxx_header = "REU/SimObjects/cache_info.hh"
    cxx_class = "gem5::CacheInfo"

